WISE-Community/WISE-Client
# feat(accessibility): Ensure links have sufficient contrast

## Changes
- Darkened primary and accent colors so they have sufficient contrast for readability on light backgrounds.
- Added underline to link text to distinguish links from surrounding text.
- Changed color and added keyboard accessibility to some links in the site.

A nice resource for checking contrast for accessibility: [Contrast Checker](https://webaim.org/resources/contrastchecker/).

Resolves #1392.

## Test
- Make sure regular links have underlines across apps.
- Make sure you can use tab and enter key to interact with links on Teacher class schedule and unit libraries.
